account_avatax/models/account_tax.py

```diff
@@ -41,7 +42,22 @@ def get_avalara_tax(self, tax_rate, doc_type):
                 tax.name = self._get_avalara_tax_name(tax_rate, doc_type)
             return tax
         else:
-            tax = self.env.ref("account_avatax.avatax", raise_if_not_found=False)
+            # Get tax based on company
+            tax = self.search([
+                ("is_avatax", "=", True),
+                ("name", "=", "AVATAX"),
```

Comment:
instead of name search, can we use Avatax record XML id? (Ref https://github.com/OCA/account-fiscal-rule/blob/14.0/account_avatax/data/avalara_salestax_data.xml#L12)